cockroachdb/assets/service_checks.json

```
@@ -1 +1,32 @@
-[]
+[
+  {
+    "agent_version": "7.34.0",
+    "integration": "CockroachDB",
+    "check": "cockroachdb.openmetrics.health",
+    "statuses": [
+      "ok",
+      "critical"
+    ],
+    "groups": [
+      "host",
+      "endpoint"
+    ],
+    "name": "CockroachDB openmetrics endpoint up",
+    "description": "Returns `CRITICAL` if the check cannot access the openmetrics metrics endpoint of CockroachDB."
+  },
+  {
+    "agent_version": "7.19.0",
+    "integration": "CockroachDB",
+    "check": "cockroachdb.prometheus.health",
+    "statuses": [
+      "ok",
+      "critical"
+    ],
+    "groups": [
+      "host",
+      "endpoint"
+    ],
+    "name": "CockroachDB prometheus endpoint up",
+    "description": "Returns `CRITICAL` if the check cannot access the openmetrics metrics endpoint of CockroachDB."
```

```suggestion
    "description": "Returns `CRITICAL` if the check cannot access the Prometheus metrics endpoint of CockroachDB."
```

I think there was a copy/paste error here